RadixWallet/Clients/DeepLinkHandlerClient/DeepLinkHandlerClient+Interface.swift

```diff
@@ -0,0 +1,21 @@
+// MARK: - DeepLinkHandlerClient
+/// A client that manages the handling of the received deepLink.
+/// It will delegate the deepLink to the respective subcomponent.
+public struct DeepLinkHandlerClient: DependencyKey {
```

Comment:
Hmm should client be `Sendable`?

Reply:
Not needed, it is used only from the main thread really. I could add it for the sake of it, but it will not be useful.